Modules/AuctionManager/GUI.lua

```diff
@@ -89,6 +89,9 @@ local function CreateBidWindow(self)
         if selected.cols == nil then return false end -- Handle column titles click
         self.awardPlayer = selected.cols[1].value or ""
         -- self.awardValue = selected.cols[4].value or 0
+        if not self.awardValue or self.awardValue == '' then
+            AuctionManagerGUI:UpdateBids(self)
```

Comment:
Syntactic sugar: `Array:Method()` is same as `Array.Method(self)` thus we don't pass `self` when using `:`